delete_item.go

```diff
@@ -16,11 +15,10 @@ import (
  * @return true if the record was deleted, false otherwise
 */
 func (t *Client) DeleteItem(ctx context.Context, pk string, sk string) error {
-	table := os.Getenv("DYNAMODB_TABLE")
 
 	//delete item from dynamodb
 	input := &dynamodb.DeleteItemInput{
-		TableName: &table,
+		TableName: &t.TableName,
```

Comment:
how about other function @thusharprakash ?

Reply:
Only this function reads env variable @quangthanh0909 
![image](https://github.com/oolio-group/dynago/assets/4880359/5402cf4c-73c6-4cc3-8056-35e7a309f58d)